### PR TITLE
feat(comments): use disqus as first comment section

### PR DESCRIPTION
**Pull request by Slashgear**
Blogs often have comment section.

Disqus free mode is great (except tracking part). It should to it for now.

We can imagine using another repository issues to handle comment. Author would just have to define an Issue ID in frontmatter. A dedicated component would load comments on issues and display them in page.

### CODE REVIEW

File: docs/.vuepress/components/Disqus.vue [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<template>
+  <div id="disqus_thread"></div>
+</template>
+<script>
+export default {
+  mounted() {
+    var disqus_config = function() {
```
Comment (mfrachet) [inline docs/.vuepress/components/Disqus.vue:7]: have you really used a `var` bro? XD

File: docs/.vuepress/components/Disqus.vue [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+      this.page.url = window.location.origin;
```
Comment (mfrachet) [inline docs/.vuepress/components/Disqus.vue:8]: Et je sais pas si c'est voulu, mais `this.page` est rattaché à ta fonction et non pas a ton composant

File: docs/.vuepress/components/Disqus.vue (hunk 1 above)
Comment (Slashgear) [reply docs/.vuepress/components/Disqus.vue:7]: This should be global for disqus

File: docs/.vuepress/components/Disqus.vue (hunk 2 above)
Comment (Slashgear) [reply docs/.vuepress/components/Disqus.vue:8]: Yep c'est voulu

**Comment by Slashgear** [comment]
Et toi @Swiip C'est quoi ton avis la dessus ?

File: docs/.vuepress/components/Disqus.vue (hunk 1 above)
Comment (mfrachet) [reply docs/.vuepress/components/Disqus.vue:7]: 🤦‍♂️

**Review by mfrachet — APPROVED** [review]

File: docs/.vuepress/components/Disqus.vue (hunk 1 above)
Comment (Osirisxxl) [reply docs/.vuepress/components/Disqus.vue:7]: global as in window global ?
Using var would not have `disqus_config` be available globally right ? 🤔 

Does it work as intended at the moment ?

**Comment by Osirisxxl** [comment]
Your suggestion to use another repo to handle comment looks intriguing and appealing.
Though I don't see the full path for it.

In case this is your final solution, there will be the need to migrate disqus comments to the issue solution. Don't know much about how to solve this, but it is worth noting anyway.
Would disqus comments across posts be easily accessible ?

**Comment by Swiip** [comment]
A few thoughts on the matter, I wish I could do more.

- Privacy and publication process should be an important matter here as the whole purpose of the plateforme is to remove control and ownership on articles published. If Discuss owns the data, it could really be a shame but if we use GitHub, it's GitHub which owns the data and the tracking, is that really better?
- Idea of a second repo seems strange to me. This repo is where you have to submit a PR to add an article, issues to exchange on them should be the same one. Of course it would be difficult to sort issues on articles and others on the website itself.
- Technically, using GitHub issues, we would have to write components to request GitHub API to show comments directly below the article, I'm not sure we are able to do this simply and quickly. Before going further, is there a "no auth" access to issue comments in GitHub API?

**Comment by Slashgear** [comment]
Personally I would trust Github better than Disqus.
Playing with two repo could be tricky yeah 😄 

V3 Rest API is "no auth" to read, example here.

https://api.github.com/repos/open-blog-initiative/open-blog.dev/issues/4

**Pull request closed without merging**